src/clj/game/cards/hardware.clj

```diff
@@ -614,6 +614,18 @@
     :recurring (effect (set-prop card :rec-counter (count (:ices (get-in @state [:corp :servers :hq])))))
     :effect (effect (set-prop card :rec-counter (count (:ices (get-in @state [:corp :servers :hq])))))}
 
+   "Mâché"
+   {:abilities [{:label "Draw 1 card"
+                 :msg "draw 1 card"
+                 :counter-cost [:power 3]
+                 :effect (effect (draw :runner 1))}]
+    :events {:runner-trash {:once :per-turn
+                            :async true
+                            :req (req (and (card-is? target :side :corp)
```

Comment:
How restrictive is the `:runner-trash` event? Does it only fire when card is accessed?

I'm wondering if this will trigger if you use PolOp, Apocalypse or Keyhole to trash a card.

Might want to add that case to the test as well.

Reply:
Ah yikes, you're probably right. Hmm, I'll have to think about how to handle this.

Reply:
In theory, `(:access @state)` should only be true while a card is being accessed. In theory.